testsuite/cli/src/client_proxy.rs

```diff
@@ -618,11 +623,18 @@ impl ClientProxy {
                 .get_txn_by_acc_seq(account, sequence_number - 1, true)
             {
                 Ok(Some(txn_view)) => {
-                    println!("transaction is stored!");
-                    if txn_view.events.is_empty() {
-                        println!("no events emitted");
+                    if txn_view.vm_status != StatusCode::EXECUTED {
+                        break Err(format_err!(
+                            "transaction failed to execute; status: {:?}!",
+                            txn_view.vm_status
+                        ));
+                    } else {
+                        println!("transaction is stored!");
```

Comment:
I think you meant transaction is executed.

Reply:
@phoenix-antigravity changed it from "completed" to "stored" in c433798c52. I'm just moving this code. But, maybe you have a point -- since we now know that the execution was successful, maybe the message should change as well. Phoenix, WDYT?

Reply:
I'll go ahead and change it to "transaction executed". I'm also going to invert the comparison and put the successful case first in the if-statement.